### ISSUE TITLE
Code coverage CI runs are failing

### ISSUE THREAD

**Issue by JonHanna** [dotnet/corefx#14703]
Code coverage runs are failing with commands along the lines of:

```
D:\j\workspace\code_coverage---fc4f7f0d\Tools\CodeCoverage.targets(82,5): error MSB3073: The command "D:\j\workspace\code_coverage---fc4f7f0d\packages/ReportGenerator\2.5.0\tools\ReportGenerator.exe -targetdir:D:\j\workspace\code_coverage---fc4f7f0d\bin/tests/coverage\ -reporttypes:Html;Badges -reports:D:\j\workspace\code_coverage---fc4f7f0d\bin/tests/coverage\\*.coverage.xml" exited with code 1. [D:\j\workspace\code_coverage---fc4f7f0d\build.proj
```

The last successful run was on the 12th.

**Comment by karelz** [comment]
@crummel do you know what is going on?

**Comment by crummel** [comment]
It looks like coverage tests are not actually being run, so report generation is failing with no input reports.  I tried this out with manually passing the internal coverage flags and didn't have any luck, so it looks like a deeper issue than just build.cmd/config.json stuff.  I'll continue looking into it.

**Comment by crummel** [comment]
The issue looks like it was introduced somewhere in the range of:
1fd0ce0834978d4a1ffe52b9a2c0b0fdb52ec475
fba133c923d53932285b77581f4d5336fddffd38
89cd1913d1faf159bde3b83e28e96c4962091f37
@karajas any idea if you changed something that might affect coverage?

**Comment by karajas** [comment]
I hadn't tested coverage with the changeset above, but I believe dev/eng may be merging soon into master and we should have coverage working on that branch.

**Comment by stephentoub** [comment]
@crummel, @karajas, any updates on this?  Have we confirmed that code coverage runs are working in dev/eng?

**Comment by crummel** [comment]
@karajas @stephentoub I just tried it and I'm seeing the same issue in dev/eng.

**Comment by karajas** [comment]
Opened an issue in dev/eng https://github.com/dotnet/corefx/issues/15126 .

**Comment by JonHanna** [comment]
Fixed by #15306
